tests/functional/api/test_gitlab.py

```diff
@@ -217,8 +217,8 @@ def test_list_all_false_nowarning(gl, recwarn):
 
 
 def test_list_all_true_nowarning(gl, recwarn):
-    """Using `all=True` will disable the warning"""
-    items = gl.gitlabciymls.list(all=True)
+    """Using `get_all=True` will disable the warning"""
+    items = gl.gitlabciymls.list(get_all=True)
     assert not recwarn
```

Comment:
Can we also make sure to have a test with just `all=True` to make sure we aren't breaking anything?

Reply:
Might seem a little lazy but I parametrized the existing tests to test both in https://github.com/python-gitlab/python-gitlab/pull/1458/commits/6d69d3317894bd9a4b4c289b1ed9d8a87871ca1f in all our tests where we had `all=True` before.

Still not entirely sure how I feel about this PR even after over a year. But - it fixes things! :sweat_smile: